<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manufacturer</class>
 <widget class="QDialog" name="Manufacturer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="manufacturerCode_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code  : </string>
   </property>
  </widget>
  <widget class="QLabel" name="manufacturerName_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_PushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_PushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="manufacturerName_LineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="manufacturerCode_LineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>manufacturerCode_LineEdit</tabstop>
  <tabstop>manufacturerName_LineEdit</tabstop>
  <tabstop>cancel_PushButton</tabstop>
  <tabstop>set_PushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
